integration/src/integration/java/pl/allegro/tech/hermes/integration/management/SupportTeamToOwnerMigratorIntegrationTest.java

```diff
@@ -0,0 +1,88 @@
+package pl.allegro.tech.hermes.integration.management;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pl.allegro.tech.hermes.api.OwnerId;
+import pl.allegro.tech.hermes.api.Subscription;
+import pl.allegro.tech.hermes.api.Topic;
+import pl.allegro.tech.hermes.integration.IntegrationTest;
+import pl.allegro.tech.hermes.management.migration.owner.SupportTeamToOwnerMigrator;
+
+import javax.ws.rs.core.Response;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static pl.allegro.tech.hermes.test.helper.builder.SubscriptionBuilder.subscription;
+
+public class SupportTeamToOwnerMigratorIntegrationTest extends IntegrationTest {
+
+    public static final String ALL = "{\"query\": {}}";
+
+    @BeforeMethod
+    public void clearTopics() {
+        management.query().queryTopics(ALL).forEach(topic ->
+                management.topic().remove(topic.getQualifiedName()));
+        wait.until(() -> management.query().queryTopics(ALL).isEmpty());
+    }
+
+    @Test
+    public void shouldMigrateGroupsTopicsAndSubscriptionsToOwnerModel() {
```

Comment:
this test is also barely readable because of repetitions of method calls, which could be replaced with short variable names / constants